Dockerfile

```diff
@@ -1,13 +1,31 @@
-FROM alpine
+FROM ruby:2.6-alpine
 
-RUN apk update
-RUN apk add git openssh-client curl jq
+RUN apk add \
+    curl \
+    docker \
+    git \
+    jq \
+    openssh-client
+RUN gem install bundler
 
-RUN adduser app -D
 WORKDIR /home/app
-COPY components.conf update-version update-version-wrapper ./
-RUN chmod +r components.conf
-RUN chmod +rx update-version update-version-wrapper
-USER app
```

Comment:
It's a good practice to not run as root inside the Docker image.

Reply:
I couldn't make it work since we have to run dockerd, which has some unusual and elevated demands.